apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/index.php

```diff
@@ -96,42 +73,13 @@ function enqueue_launch_flow_script_and_style( $site_launch_options ) {
 			// For redirect or invalid flows, skip & exit early.
 			return;
 	}
-
-	$asset_file          = include plugin_dir_path( __FILE__ ) . 'dist/' . $script_name . '.asset.php';
-	$script_dependencies = isset( $asset_file['dependencies'] ) ? $asset_file['dependencies'] : array();
-	$script_file         = 'dist/' . $script_name . '.js';
-	$script_version      = isset( $asset_file['version'] ) ? $asset_file['version'] : filemtime( plugin_dir_path( __FILE__ ) . $script_file );
-
-	wp_enqueue_script(
-		'a8c-fse-editor-site-launch-script',
-		plugins_url( $script_file, __FILE__ ),
-		$script_dependencies,
-		$script_version,
-		true
-	);
-
-	wp_set_script_translations( 'a8c-fse-editor-site-launch-script', 'full-site-editing' );
-
-	$style_file    = 'dist/' . $script_name . ( is_rtl() ? '.rtl.css' : '.css' );
-	$style_version = isset( $asset_file['version'] ) ? $asset_file['version'] : filemtime( plugin_dir_path( __FILE__ ) . $style_file );
-
-	wp_enqueue_style(
-		'a8c-fse-editor-site-launch-style',
-		plugins_url( $style_file, __FILE__ ),
-		array(),
-		$style_version
-	);
+	\A8C\FSE\use_webpack_assets( $script_name );
 }
 
 /**
  * Enqueue assets
  */
 function enqueue_script_and_style() {
-	// Avoid loading assets if possible.
-	if ( ! \A8C\FSE\Common\is_block_editor_screen() ) {
-		return;
-	}
```

Comment:
This is removed because the action `enqueue_block_editor_assets` implies the scripts are just gonna be loaded when in block editor right?

Reply:
Yeah, functionality should be the same as far as I can tell